Dockerfile

```diff
@@ -5,6 +5,7 @@ ADD . /usr/src/sriov-network-device-plugin
 ENV HTTP_PROXY $http_proxy
 ENV HTTPS_PROXY $https_proxy
 RUN apk add --update make && \
+    apk add build-base && \
```

Comment:
Why this package is required in Dockerfile, Is it needed to build multi-arch images?

Reply:
To avoid the error in the image
![travis error](https://user-images.githubusercontent.com/29833327/56564093-02fb7d00-65b6-11e9-9d78-513d2542433a.PNG)

Reply:
for docker upgrade, I updated the command,
`apt-get install --only-upgrade docker-ce`